homeassistant/components/plugwise/config_flow.py

```diff
@@ -108,6 +112,9 @@ async def async_step_user(self, user_input=None):
                 errors["base"] = "unknown"
             if not errors:
                 await self.async_set_unique_id(api.gateway_id)
```

Comment:
Remove this line.

Reply:
The `if not errors` one?

Reply:
This one: `await self.async_set_unique_id(api.gateway_id)`
This one is not removed, should be because we are adding the one below.

Reply:
Removed it accordingly